tests/core/creators/manual/detachedcustominterval.html

```diff
@@ -0,0 +1,26 @@
+<button id="startButton">Start Test</button>
```

Comment:
Test works good 👍🏻  I'd just add a visual timer counting down the time to create editor.